CODEOWNERS

```diff
@@ -2,13 +2,13 @@
 # approval within two weeks.
 #
 # These owners will be the default owners for everything in the repo.
-*       @kroening @tautschnig @peterschrammel @chrisr-diffblue
+*       @kroening @tautschnig @peterschrammel @chrisr-diffblue @TGWDB
 
 # These files should rarely change
 
 /src/big-int/ @kroening
-/src/ansi-c/ @kroening @tautschnig @chrisr-diffblue
-/src/assembler/ @kroening @tautschnig @chrisr-diffblue
+/src/ansi-c/ @kroening @tautschnig @chrisr-diffblue @peterschrammel @thomasspriggs @NlightNFotis @TGWDB
+/src/assembler/ @kroening @tautschnig @chrisr-diffblue @peterschrammel @thomasspriggs @NlightNFotis @TGWDB
```

Comment:
I can't say that I am any kind of authority on our assembly support, having never worked on its implementation in cbmc. However I have enough familiarity with cbmc front-ends in general, that I am not going to strenuously object to being made a codeowner of this, if we need more codeowners.

Reply:
Dare I question whether the above changes are in line with "regularly updated parts of the code?"

Reply:
This assembler sub folder has not been updated particularly regularly, as I read the commit history. Therefore adding more code owners for it is not in line with this PR's "regularly updated parts of the code" description. @TGWDB What is your view on this?

Reply:
I don't know anything about what's in `src/assembler` myself, so I don't think I'm particularly suited for being a code owner for this.

Reply:
I'll reduce some of these (commit to come shortly). I agree they may not be "regular" as the PR suggests, on the other hand we had a problem with getting a PR approved by codeowner around the `ansi-c` part, so I'll add a little more here.

Reply:
If it helps, my role has very recently changed so that I now have a lot more time to work on CBMC.